packages/esm-patient-chart-app/src/index.ts

```diff
@@ -30,20 +29,25 @@ function setupOpenMRS() {
   defineExtensionConfigSchema('nav-group', genericNavGroupConfigSchema);
   defineExtensionConfigSchema('dashboard', genericDashboardConfigSchema);
 
-  const t = i18next.getFixedT(null, moduleName);
   registerBreadcrumbs([
     {
       path: spaBasePath,
-      title: () => Promise.resolve(t('patientBreadcrumb', 'Patient')),
+      title: () => Promise.resolve(window.i18next.t('patientBreadcrumb', { defaultValue: 'Patient', ns: moduleName })),
```

Comment:
Hi @ibacher !
Is window.i18next working for you?
It's throwing error, since i18next is undefined

Reply:
Yes... I only committed this once it was working. `window.i18next` should be defined, especially by the point this gets called...

Reply:
Ohh okay 🤔 

I got the catch

Reply:
I was actually calling the `window.i18next` inside the `setupOpenMRS`, but you called it inside the breadcrumb function.

Yes, by the time breadcrumb function was called, it is setup.

Reply:
Hmmm... that's weird.

Reply:
I went through the app shell too. [Here](https://github.com/openmrs/openmrs-esm-core/blob/1acdec571b0284d0c78c367b1377d544bc87485a/packages/shell/esm-app-shell/src/run.ts#L345) it showed that apps are loading first, before the i18 is setup. (Correct me if I'm wrong)

Reply:
Thanks for the help @ibacher !
I tried a lot of ways but wasn't able to figure this out

Reply:
> it showed that apps are loading first, before the i18 is setup

Yeah, that's why I had to be done inside a function that's called later. We could rework things, but we'd have to be a little careful about it; translations are loaded [from the module's `importTranslation()` export](https://github.com/openmrs/openmrs-esm-core/blob/1acdec571b0284d0c78c367b1377d544bc87485a/packages/shell/esm-app-shell/src/locale.ts#L91) function, but we're only able to load the module once `loadModules()` is called, so we'd have to add a step between `loadApps()` and `setupApps()` to initialize the i18next context if we really need this to work inside `setupOpenMRS()`.

I'm not sure of the value of doing that, though?

Reply:
Also, `i18next` probably needs to be shared so we can avoid loading directly from `window`.